qutebrowser/browser/qutescheme.py

```diff
@@ -159,6 +161,35 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('history')
+def qute_history(_url):
+    """Handler for qute:history. Display browsing history."""
+    history = reversed(objreg.get('web-history').history_dict.values())
```

Comment:
On a second thought, this makes no sense at all, as a dictionary is unordered. You probably want something involving `sorted(..., key=..., reverse=True)`.